scripts/src/config.ts

```diff
@@ -1,19 +1,20 @@
-import { LogTarget } from "./logging";
 import { Options as SqlOptions } from "sequelize";
 
-export type DatabaseConfig = {
+import { LogTarget } from "./logging";
+
+export interface DatabaseConfig {
 	database: string;
 	username: string;
 	password: string;
 	options: SqlOptions;
-};
+}
 
-export type Config = {
+export interface Config {
```

Comment:
why change from type?

Reply:
the linter prefers interfaces over type aliases ([interface-over-type-literal](https://palantir.github.io/tslint/rules/interface-over-type-literal/)), because of [this part of the docs](https://www.typescriptlang.org/docs/handbook/advanced-types.html):

> A second more important difference is that type aliases cannot be extended or implemented from (nor can they extend/implement other types). Because an ideal property of software is being open to extension, **you should always use an interface over a type alias if possible**.